src/igvfd/schemas/biomarker.json

```diff
@@ -0,0 +1,75 @@
+{
+    "title": "Biomarker",
+    "description": "Schema for submitting a biomarker.",
+    "$id": "/profiles/biomarker.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "name",
+        "quantification"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "name_quantification"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        }
+    ],
+    "dependentSchemas": {},
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "name": {
+            "title": "Name",
+            "description": "The biomarker name.",
+            "comment": "Only admins are allowed to set this value.",
+            "type": "string",
+            "permission": "import_items"
+        },
+        "classification": {
+            "title": "Biomarker Classification",
```

Comment:
I think the tilte could be changed to Classification (no need for Biomarker mentioning in the title)

Reply:
done